kickstarts/partials/main/disk_layout.ks.erb

```diff
@@ -2,17 +2,18 @@
 zerombr
 clearpart       --all --drives=vda
 
-part pv.1             --ondrive=vda                     --size=<%= @target == "azure" ? "11264" : "16384" %>
-part /boot            --ondrive=vda                     --size=512   --fstype=xfs
-part /var/www/miq_tmp --ondrive=vda                     --size=10240 --fstype=xfs --fsoptions="rw,noatime,nobarrier"
+part pv.1             --ondrive=vda                   --size=<%= @target == "azure" ? "21264" : "26384" %>
```

Comment:
The original number was base 1024.  Can we either stick to 1024, or fix the size correctly if 1000 is what we should be using? I think mixing the two will be confusing...